src/lib/BLE/devBLETele.cpp

```diff
@@ -0,0 +1,264 @@
+#if defined(PLATFORM_ESP32)
+
+#include "CRSF.h"
+#include "FHSS.h"
+#include "NimBLEDevice.h"
+#include "POWERMGNT.h"
+#include "common.h"
+#include "hwTimer.h"
+#include "logging.h"
+#include "options.h"
+
+#include "devBLETele.h"
+
+#include "config.h"
+extern TxConfig config;
+
+NimBLEServer *pServer;
+NimBLECharacteristic *rcCRSF;
+
+#define BLE_UPDATE_RATE 500
```

Comment:
If this is going to be "configurable", then rather than a define it should be added the options so it can configured in the web UI, and the default value can be added to `user_defines.txt`